test/acceptance/test/profiles_test.go

```diff
@@ -104,6 +103,26 @@ podinfo	Podinfo Helm chart for Kubernetes	6.0.0,6.0.1
 		Expect(err).NotTo(HaveOccurred())
 		Expect(string(values)).To(ContainSubstring("# Default values for podinfo"))
 	})
+	It("profiles are installed into a different namespace", func() {
```

Comment:
can we just make this the default behaviour of the above test, rather than a new test? We don't need to test same-namespace works if cross-namespace works

Reply:
Does that work? I mean, I'm not sure, I have to look at the test code, but it installs wego in one namespace, but the install requires the helmRepository in order for getProfiels to work to see the helmrepository. `GetProfiles` still needs a namespace and repo to work, and we aren't providing that as a setup value right now. We haven't yet made it work with all HelmRepositories...

Reply:
Yeah, that won't work, because the install sets it to that namespace + `weavework-charts` so if I run this with the other namespace it fails since there are no profiles because it's not in the expected namespace.

For now, this is fine, once we make all of this work without the need of specifying the namespace and the name of the helmrepository, we can update this test to no longer care. :)

Reply:
:+1: